tests/asserts/final/publication/geoserver.py

```diff
@@ -140,3 +140,13 @@ def wms_bbox(workspace, publ_type, name, *, exp_bbox, crs, precision=0.00001, co
     assert_util.assert_same_bboxes(exp_bbox, bbox, precision)
     if contains:
         assert bbox_util.contains_bbox(bbox, exp_bbox, precision=precision / 10000)
+
+
+def wms_all_styles_have_legend_url(workspace, publ_type, name):
```

Comment:
Let's assert in workspace_wms_1_3_0_capabilities_available that all styles have LegendURL. Probably renamt the method.

Ale parametrize the method with WMS versions (1.3.0 and 1.1.1].